cypress-shared/support/fedex-shipping/graphql_testcase.js

```diff
@@ -29,25 +35,33 @@ export function saveAppSetting(appDatas, allSla) {
   if (allSla) {
     appDatas.slaSettings = allSla
   }
+  cy.qe('Save App setting via graphql.')
 
   const query =
     'mutation' +
     '($userCredentialKey: String, $userCredentialPassword: String, $defaultDeliveryEstimateInDays: String, $parentCredentialKey: String, $parentCredentialPassword: String, $clientDetailMeterNumber: String, $clientDetailAccountNumber: String, $isLive: Boolean, $residential: Boolean,$optimizeShippingType: Int,$unitWeight: String,$unitDimension: String,$packingAccessKey: String,$slaSettings:[SlaSettingsInput])' +
     '{saveAppSetting(appSetting: {userCredentialKey:$userCredentialKey,userCredentialPassword:$userCredentialPassword,defaultDeliveryEstimateInDays:$defaultDeliveryEstimateInDays,parentCredentialKey:$parentCredentialKey,parentCredentialPassword:$parentCredentialPassword,clientDetailMeterNumber:$clientDetailMeterNumber,clientDetailAccountNumber:$clientDetailAccountNumber,isLive:$isLive,residential:$residential,optimizeShippingType:$optimizeShippingType,unitWeight:$unitWeight,unitDimension:$unitDimension,packingAccessKey:$packingAccessKey,slaSettings:$slaSettings})' +
     '@context(provider: "vtexus.fedex-shipping")}'
 
+  cy.qe(`The graphQl mutation is,${query}`)
+  cy.qe(`Variables - ${appDatas}`)
+
   return {
     query,
     queryVariables: appDatas,
   }
 }
 
 export function savePackingOptimizationAppSetting(settings) {
+  cy.qe('Save packing optimization app settings via graphql.')
+
   const query =
     'mutation' +
     '($accessKey: String, $containerList: [ContainerInput])' +
     '{saveAppSetting(appSetting: {accessKey:$accessKey,containerList:$containerList})' +
     '@context(provider: "vtex.packing-optimization")}'
+  cy.qe(`The graphql mutation is,${query}`)
+  cy.qe(`Variables - ${settings}`)
```

Comment:
@thyarles suggested to not print variables since it may have sensitive information

Reply:
Added that condition